contracts/TokenFRTProxy.sol

```diff
@@ -0,0 +1,39 @@
+pragma solidity ^0.4.24;
+
+import "@gnosis.pm/util-contracts/contracts/Proxy.sol";
+import "@gnosis.pm/util-contracts/contracts/GnosisStandardToken.sol";
+
+contract TokenFRTProxy is Proxy, GnosisStandardToken {
+    ///@dev State variables remain for Block explorer compatibility
+    string public constant symbol = "MGN";
+    string public constant name = "Magnolia Token";
+    uint8 public constant decimals = 18;
+
+    struct unlockedToken {
```

Comment:
I think we don't need the variables from line 12 to line 27, right?

Reply:
I'm not sure - I think we don't since the order for memory starts the same - I think anything after the initial variables isn't required, wdyt @josojo?

Reply:
moved `owner` to top & removed other logic